develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/03-upgrading-the-liferay-maven-environment.markdown

```diff
@@ -0,0 +1,107 @@
+        </executions>
+            <configuration>
+                <docrootDirName>src/main/webapp</docrootDirName>
+            </configuration>
+    </plugin>
+
+Some popular Liferay Maven plugins are listed below, with their corresponding
+artifact IDs and tutorials explaining how to configure them:
+
+**Popular Liferay Maven Plugins**
+
+Name | Artifact ID | Tutorial |
+:----| :---------- | :------- |
+CSS Builder | [com.liferay.css.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.css.builder) | [Compiling SASS Files in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/compiling-sass-files-in-a-maven-project) |
+Lang Builder | [com.liferay.lang.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.lang.builder) | Coming Soon |
+Service Builder | [com.liferay.portal.tools.service.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.portal.tools.service.builder) | [Using Service Builder in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/using-service-builder-in-a-maven-project) |
+Theme Builder | [com.liferay.portal.tools.theme.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.portal.tools.theme.builder) | [Building Themes in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/building-themes-in-a-maven-project) |
+
+Awesome! You've learned about the new Maven plugins available to you for
+@product-ver@ development. Next, you'll learn about upgrading your Liferay Maven
+artifacts.
+
+## Upgrading Liferay Maven Artifacts
+
+Many Liferay Portal 6.2 artifact dependencies you were using have changed in
+@product-ver@. See the table below for popular Liferay Maven artifacts that have
+changed:
+
+Liferay Portal 6.2 Artifact ID | @product-ver@ Artifact ID |
+:----------------------------- | :------------------------ |
+`portal-service` | `com.liferay.portal.kernel` |
+`util-bridges` | `com.liferay.util.bridges` |
+`util-java` | `com.liferay.util.java` |
+`util-slf4j` | `com.liferay.util.slf4j` |
+`util-taglib` | `com.liferay.util.taglib` |
+
+For more information on resolving dependencies in @product-ver@, see the
+[Resolving a Plugin's Dependencies](/develop/tutorials/-/knowledge_base/7-0/resolving-a-plugins-dependencies)
+tutorial.
+
+Of course, you must also update the artifacts you're referencing for your
+projects. If you're using the Central Repository to to install Liferay Maven
+artifacts, you won't need to do anything further than updating the artifacts in
+your POMs. If you're, however, working behind a proxy or do not have internet
+access, you must update your company-shared or local repository with the latest
+@product-ver@ Maven artifacts. See the
+[Installing Liferay Maven Artifacts](/develop/tutorials/-/knowledge_base/7-0/installing-liferay-maven-artifacts)
+tutorial for instructions.
+
+With these updates, you can easily upgrade your Liferay Maven Environment so you
+can begin developing traditional plugin projects for @product-ver@.
```

Comment:
Sure thing! Working on those docs now, and will reference them when they're published. 😃